ztp/.gitignore

```diff
@@ -1,2 +1,3 @@
-ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator/PolicyGenerator
+ztp-policy-generator/kustomize/plugin/policyGenerator/v1/policygenerator
+ztp-policy-generator/policygenerator/PolicyGenerator
```

Comment:
What about moving this up more level, to `ztp/policyGenerator/`?  The go path would look a lot less redundant that way!